frontend/sozisel-app/src/hooks/useFetchEventLiveResult.ts

```diff
@@ -0,0 +1,140 @@
+import {
+  EventResultSubmittedSubscription,
+  QuizParticipantSummary,
+  useEventResultSubmittedSubscription,
+  useQuizParticipantsSummaryQuery,
+} from "../graphql";
+import { useEffect, useState } from "react";
+
+type QuizParticipantsSummary = ({
+  __typename?: "QuizParticipantSummary" | undefined;
+} & Pick<QuizParticipantSummary, "numberOfPoints" | "fullName">)[];
+
+interface EventResult {
+  pointSum: number;
+  completedTrialsNumber: number;
+  scoresDistribution: ScoreDistribution[];
+}
+
+interface ScoreDistribution {
+  score: number;
+  counter: number;
+}
+
+type Typename = "QuizSimpleResult";
+
+const addNewScore = (
```

Comment:
All those functions calculating scores does not belong in this file which does not specify that is is mainly for quiz results. I understand if you keep fetching the quizes here but not putting 3 functions mainly for calculating scores (correct me if those functions are meant for every even type but I don't think so)